### PR TITLE
Switch from FFF to epoch

### LINKED ISSUE #1843: [Feedback] Add content about using the Unix epoch to caggs HowTo on page: /timescaledb/latest/how-to-guides/continuous-aggregates/time/
### Is it easy to find the information you need?

None

### Are the instructions clear?

None

### How could we improve the Timescale documentation site?

why is this document using FFF as an example? it would be helpful if we didn't have to figure out how to do this with the commonly used Unix epoch timestamp through trial and error. it strikes me that this would be more practical than a quirky example

### PR DESCRIPTION
**Pull request by Loquacity**
# Description

Updates the time section in the caggs How To to use Unix Epoch rather than the joke FFF system.

# Links

Fixes https://github.com/timescale/docs/issues/1843

# Writing help

For information about style and word usage, see the [style guide](https://docs.timescale.com/timescaledb/latest/contribute-to-docs/)

# Review checklists
Reviewers: use this section to ensure you have checked everything before approving this PR:

## Subject matter expert (SME) review checklist

- [ ] Is the content technically accurate?
- [ ] Is the content complete?
- [ ] Is the content presented in a logical order?
- [ ] Does the content use appropriate names for features and products?
- [ ] Does the content provide relevant links to further information?

## Documentation team review checklist

- [ ] Is the content free from typos?
- [ ] Does the content use plain English?
- [ ] Does the content contain clear sections for concepts, tasks, and references?
- [ ] Have any images been uploaded to the correct location, and are resolvable?
- [ ] If the page index was updated, are redirects required
      and have they been implemented?
- [ ] Have you checked the built version of this content?

### CODE REVIEW

File: timescaledb/how-to-guides/continuous-aggregates/time.md [hunk 1]
```diff
@@ -85,26 +83,33 @@ minutes.
 
     ```sql
     SELECT create_hypertable('devices', 'time',
-      chunk_time_interval => 1000);
+      chunk_time_interval => 10);
```
Comment (bithead2k) [inline timescaledb/how-to-guides/continuous-aggregates/time.md:86]: I meant to change this to 20 minutes, and the time_bucket to 10.

File: timescaledb/how-to-guides/continuous-aggregates/time.md [hunk 2]
```diff
@@ -85,26 +83,33 @@ minutes.
...
-1.  At the `psql` prompt, set up a function to convert the time to the FFF system:
+1.  At the `psql` prompt, set up a function to convert the time to the Unix epoch:
 
     ```sql
-    CREATE FUNCTION current_microfortnight() RETURNS BIGINT
+    CREATE FUNCTION current_epoch() RETURNS BIGINT
     LANGUAGE SQL STABLE AS $$
-    SELECT CAST(1209600 * EXTRACT(EPOCH FROM CURRENT_TIME) / 1000000 AS BIGINT)
-     $$;
+    SELECT EXTRACT(EPOCH FROM CURRENT_EPOCH)::bigint;
```
Comment (bithead2k) [inline timescaledb/how-to-guides/continuous-aggregates/time.md:110]: Need the closing quotes after the function:
SELECT EXTRACT(EPOCH FROM CURRENT_EPOCH)::bigint;
$$

**Review by bithead2k — CHANGES_REQUESTED** [review]
All the verbiage looks good and the edits are sane.  Just need the 3 simple changes.

File: timescaledb/how-to-guides/continuous-aggregates/time.md (hunk 1 above)
Comment (Loquacity) [inline timescaledb/how-to-guides/continuous-aggregates/time.md:86]:
```suggestion
      chunk_time_interval => 20);
```

File: timescaledb/how-to-guides/continuous-aggregates/time.md (hunk 2 above)
Comment (Loquacity) [inline timescaledb/how-to-guides/continuous-aggregates/time.md:110]:
```suggestion
    SELECT EXTRACT(EPOCH FROM CURRENT_EPOCH)::bigint;$$;
```

**Review by Rajakavitha1 — APPROVED** [review]

**Review by bithead2k — APPROVED** [review]
All set AFAICT

**Pull request merged**